frontend/test/metabase/scenarios/admin/people/people.cy.spec.js

```diff
@@ -358,6 +356,87 @@ describeEE("scenarios > admin > people", () => {
     cy.findByText("Question").should("not.exist");
     cy.findByText("Dashboard").should("not.exist");
   });
+
+  describe("email configured", () => {
+    beforeEach(() => {
+      // Setup email server, since we show different modal message when email isn't configured
+      cy.request("PUT", "/api/setting", {
+        "email-from-address": "admin@metabase.test",
+        "email-from-name": "Metabase Admin",
+        "email-reply-to": "reply-to@metabase.test",
+        "email-smtp-host": "localhost",
+        "email-smtp-password": null,
+        "email-smtp-port": "1234",
+        "email-smtp-security": "none",
+        "email-smtp-username": null,
+      });
```

Comment:
```suggestion
      setupSMTP();
```

https://github.com/metabase/metabase/blob/master/frontend/test/__support__/e2e/helpers/e2e-email-helpers.js#L13:L29

Reply:
Wow many thanks.

Reply:
Welp, I just realized the `setupSMTP` is even already imported in this file 🤦